readme.md

```diff
@@ -81,7 +85,8 @@ Your Rails database has two tables. `students` has the columns `id` and `name`,
 Use ActiveRecord to create a new `pbj` sandwich and make it belong to the student named Geraldo.
 
 ```rb
-# Your answer...
+geraldo = Student.create(name: "Geraldo")
+Sandwich.create(type: "pbj", student_id: Student.find_by(name: "Geraldo").id)
```

Comment:
You already have "geraldo", use them in this statement.
Using your code that would be `Sandwich.create(type: "pbj", student: geraldo).
I recommend:
```ruby
gerold.sandwiches.create(type: "pbj")
```
Note: "type" is a reserved column name in ActiveRecord.  We should avoid using it.